docs/quickstart.rst

```diff
@@ -77,35 +78,52 @@ In order to run the code samples in the Quickstart tutorial, you need to install
 Loading the dataset
 ^^^^^^^^^^^^^^^^^^^
 
-For this example we use the
-`UCI adult dataset <https://archive.ics.uci.edu/ml/datasets/Adult>`_ where the
-objective is to predict whether a person makes more (label 1) or less (0)
-than $50,000 a year.
+For this example, we use a `clincial dataset <https://archive.ics.uci.edu/dataset/296/diabetes+130-us+hospitals+for+years+1999-2008>`_ 
+of hospital re-admissions over a ten-year period (1998-2008) for 
+diabetic patients across 130 different hospitals in the U.S. This scenario 
+builds upon prior research on how racial disparities impact health care 
+resource allocation in the U.S. For an in-depth analysis of this dataset,
+review the `Scipy tutorial <https://github.com/fairlearn/talks/tree/main/2021_scipy_tutorial>`_
+that the Fairlearn team presented in 2021.
+
+We will use machine learning to predict whether an individual in the dataset 
+is readmnitted to the hospital within 30 days of hospital release. 
+A hospital readmission within 30 days can be viewed as a proxy that the 
+patients needed more assistance at the release time.
+In the next section, we build a classification model to accomplish the 
+prediction task.
 
 .. doctest:: quickstart
 
     >>> import numpy as np
     >>> import pandas as pd
     >>> import matplotlib.pyplot as plt
-    >>> from fairlearn.datasets import fetch_adult
-    >>> data = fetch_adult(as_frame=True)
-    >>> X = pd.get_dummies(data.data)
-    >>> y_true = (data.target == '>50K') * 1
-    >>> sex = data.data['sex']
-    >>> sex.value_counts()
-    sex
-    Male      32650
-    Female    16192
-    Name: count, dtype: int64
+    >>> data = fetch_diabetes_hospital(as_frame=True)
+    >>> keep_idx = (data.data['gender'] == "Male") | (data.data['gender'] == "Female")
```

Comment:
If we're not evaluating by gender we can probably drop this part.

Reply:
Done!